classes/user_cat_allocation.php

```diff
@@ -56,6 +56,10 @@ protected static function define_properties() {
             'category_mode' => array(
                 'type' => PARAM_INT,
                 'message' => new \lang_string('invalidmode', 'block_evasys_sync')
+            ),
+            'standard_time_mode' => array (
+                'type' => PARAM_INT,
```

Comment:
Since we will alter this field anyways, it is not relevant, but:
Why are you not using PARAM_BOOL as a type, since you want to save a boolean value?